prusti-viper/src/encoder/middle/core_proof/adts/interface.rs

```diff
@@ -114,10 +114,16 @@ pub(in super::super) trait AdtsInterface {
     /// and its result is used to guard the injectivity property. This is
     /// intended to be used by the snapshot encoder to supply call to the
     /// validity function.
+    ///
+    /// If `generate_top_down_injectivity_axiom` is true, do not generate
```

Comment:
Shouldn't it be:
```suggestion
    /// If `use_main_constructor_destructors` is true, do not generate
```

Reply:
I think that must've been one of your commits that got mixed in when I squashed the commits.